File: tests/human/x/cs/user_type_literal.cs
using System;

class Person
{
    public string name;
    public int age;
}

class Book
{
    public string title;
    public Person author;
}

class Program
{
    static void Main()
    {
        var book = new Book
        {
            title = "Go",
            author = new Person { name = "Bob", age = 42 },
        };
        Console.WriteLine(book.author.name);
    }
}

File: tests/human/x/elixir/user_type_literal.ex
defmodule Person do
  defstruct [:name, :age]
end

defmodule Book do
  defstruct [:title, :author]
end

book = %Book{
  title: "Go",
  author: %Person{name: "Bob", age: 42}
}

IO.puts(book.author.name)
